packages/forge2d/lib/src/dynamics/fixture.dart

```diff
@@ -86,53 +84,49 @@ class Fixture {
 
   double get density => _density;
 
-  /// Test a point for containment in this fixture. This only works for convex shapes.
+  /// Test a point for containment in this fixture. This only works for convex
+  /// shapes.
   ///
-  /// @param p a point in world coordinates.
-  /// @return
-  bool testPoint(final Vector2 p) {
-    return shape.testPoint(body.transform, p);
+  /// [point] should be in world coordinates.
+  bool testPoint(final Vector2 point) {
```

Comment:
why this one particular parameter is final and nothing else?

Reply:
No idea, nothing that comes from this PR though 😅